src/engine.rs

```diff
@@ -143,6 +143,9 @@ pub struct Reedline {
     hinter: Option<Box<dyn Hinter>>,
     hide_hints: bool,
 
+    // Auto-show completion menu as you type (IDE/fish style)
```

Comment:
You say "IDE" here, but looking at your code, it won't necessarily pick the `IdeMenu`, it'll pick whichever menu is named `completion_menu`. I think you just happened to see the IDE menu because that's what you have set in your Nushell config.

But also, let's not restrict this feature to the IDE menu. I prefer the columnar menu myself and would like to use that. So you can just remove the "(IDE/fish style)" bit from this comment